pkg/binding/controller.go

```diff
@@ -455,7 +455,7 @@ func (c *Controller) handleObject(obj any, resource string, eventType string) {
 		obj = typed.Obj
 		wasDeletedFinalStateUnknown = true
 	}
-	c.logger.Info("Got object event", "eventType", eventType,
+	c.logger.V(3).Info("Got object event", "eventType", eventType,
```

Comment:
I would make these V levels even higher. At least 4, maybe 5.

Reply:
Changed to 4.